test-clients/src/main/java/com/hedera/services/bdd/suites/contract/ContractCallSuite.java

```diff
@@ -147,7 +150,8 @@ public boolean canRunAsync() {
 //				smartContractInlineAssemblyCheck(), //pass
 //				ocToken(),
```

Comment:
- I think that we should address all of the commented tests and verify whether they are passing or they have proper assets. 
- I think that we need to break down the `ContractCallSuite` into a couple more suites, concentrated on the functionality and not the generic `contract call op` as it gives little to no information.